test/spec/views.js

```diff
@@ -35,6 +41,9 @@ function setup(testModule, globalOptions) {
   // module.
   testModule.origFetch = Backbone.Layout.prototype.fetchTemplate;
 
+  // keep a reference to the current utility library for mocking
+  testModule._ = globalOptions._ || _;
```

Comment:
I needed this to ensure I add a spy to the appropriate library in the two tests I added. With the all tests running in both lodash and underscore now, that spy can probably go away.